### PR TITLE
Feature/apm 332220 change validation in installations scripts to accept only services with featuresets

### PR DESCRIPTION
**Pull request by Dawid-Kaszubski**

### CODE REVIEW

**Comment by karolkieglerski** [comment]
looks ok to me, but please squash commits before merging it :)

**Review by karolkieglerski — APPROVED** [review]

File: scripts/setup.sh [hunk 1]
```diff
@@ -572,12 +567,10 @@ if [[ "$UPGRADE_EXTENSIONS" != "Y" && -n "$EXTENSIONS_FROM_CLUSTER" ]]; then
   done
 fi
 
-# Add '/default' to service name when featureSet is missing
```
Comment (mswiatkowska) [inline scripts/setup.sh:575]: Why did you remove it? Shouldn't we change `default` to `default_metrics` here?
Line 586 should also be change anyway.

Comment (Dawid-Kaszubski) [reply scripts/setup.sh:584]: We don't want to add any featureSet by default. FeatureSet for service should be defined explicitly.

Comment (Dawid-Kaszubski) [reply scripts/setup.sh:584]: We don't want to add any featureSet by default. FeatureSet for service should be defined explicitly.
